<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitorbuttons</class>
 <widget class="QWidget" name="monitorbuttons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>monitorbuttons</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="but1">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="but2">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="but3">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
